Add new CLI commands (#29495)
dotnet/docs: docs/machine-learning/tutorials/sentiment-analysis-cli.md

    ![VS solution generated by the CLI](./media/mlnet-cli/mlnet-cli-solution-explorer.png)

    > [!NOTE]
    > In the tutorial, we suggest using Visual Studio, but you can also explore the generated C# code (two projects) with any text editor and run the generated console app with the `dotnet CLI` on a macOS, Linux, or Windows machine.

    - The generated **console app** contains execution code that you must review and then you usually reuse the 'scoring code' (code that runs the ML model to make predictions) by moving that simple code (just a few lines) to your end-user application where you want to make the predictions.
    - The generated **mbconfig file** is a config file that can be used to retrain your model, either through the CLI or through Model Builder. This will also have two code files associated with it and a zip file.
        - The **training** file contains the code to build the model pipeline using the ML.NET API.
        - The **consumption** file contains the code to consume the model.
        - The **zip** file that is the generated model from the CLI.

> 1. Open the **SampleClassification.consumption.cs** file within the **mbconfig** file. You will see that there input and output classes classes which are 'data classes' or POCO classes used to hold data. It is 'boilerplate code' but useful to have it generated if your dataset has tens or even hundreds of columns.

Reviewer: ```suggestion
1. Open the **SampleClassification.consumption.cs** file within the **mbconfig** file. You'll see that there are input and output classes. These are 'data classes' or POCO classes used to hold data. The classes contain boilerplate code that's useful if your dataset has tens or even hundreds of columns.
```

For more advanced learning purposes, you can also explore the generated C# code that was used by the CLI tool to train the generated model.

> That 'training model code' is currently generated in the file generated named `SampleClassification.training.cs` so you can investigate that training code.

Reviewer: ```suggestion
That training model code is generated in the file named `SampleClassification.training.cs`, so you can investigate that training code.
```
